### DISCUSSION TITLE
Crash on MC start

### DISCUSSION THREAD

**Discussion by Freitag47** [shedaniel/RoughlyEnoughItems#982] [Q & A] [answered]
[crash-2022-07-10_14.29.46-fml.txt](https://github.com/shedaniel/RoughlyEnoughItems/files/9079041/crash-2022-07-10_14.29.46-fml.txt)

**Comment by shedaniel** [comment, marked as answer]
I have released a fix now. This is caused by new Forge releasing breaking changes.
